src/wxwidgets.mk

```diff
@@ -4,25 +4,31 @@ PKG             := wxwidgets
 $(PKG)_WEBSITE  := https://www.wxwidgets.org/
 $(PKG)_DESCR    := wxWidgets
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 3.0.2
-$(PKG)_CHECKSUM := 346879dc554f3ab8d6da2704f651ecb504a22e9d31c17ef5449b129ed711585d
+$(PKG)_VERSION  := 3.1.3
+$(PKG)_CHECKSUM := fffc1d34dac54ff7008df327907984b156c50cff5a2f36ee3da6052744ab554a
 $(PKG)_SUBDIR   := wxWidgets-$($(PKG)_VERSION)
 $(PKG)_FILE     := wxWidgets-$($(PKG)_VERSION).tar.bz2
-$(PKG)_URL      := https://$(SOURCEFORGE_MIRROR)/project/wxwindows/$($(PKG)_VERSION)/$($(PKG)_FILE)
+$(PKG)_URL      := https://github.com/wxWidgets/wxWidgets/releases/download/v$($(PKG)_VERSION)/$($(PKG)_FILE)
 $(PKG)_DEPS     := cc expat jpeg libiconv libpng sdl tiff zlib
 
 define $(PKG)_UPDATE
-    $(WGET) -q -O- 'https://sourceforge.net/projects/wxwindows/files/' | \
-    $(SED) -n 's,.*/projects/.*/\([0-9][^"]*\)/".*,\1,p' | \
-    sort -V | \
-    tail -1
+    $(WGET) -q -O- 'https://github.com/wxWidgets/wxWidgets/releases' | \
+    $(SED) -n 's,.*wxWidgets-\([0-9][^"]*\)\.tar.*,\1,p' | \
+    head -1
 endef
 
 define $(PKG)_CONFIGURE_OPTS
-        $(MXE_CONFIGURE_OPTS) \
+       --host='$(TARGET)' \
+       --build='$(BUILD)' \
+       --prefix='$(PREFIX)/$(TARGET)' \
+       $(if $(BUILD_STATIC), \
+          --disable-shared , \
+          --enable-shared )  \
```

Comment:
Why not keep using `MXE_CONFIGURE_OPTS`?

Reply:
the configure script errors out when unknown parameters are passed, i.e. --enable-static is unknown.

introduced by https://github.com/wxWidgets/wxWidgets/commit/aa7e10bb0919982bfdc510bd9cdc8b84d621d80b

Reply:
should --disable-option-checking be added instead?

i can only agree to the commit message:

"Give an error if an unknown option is specified because this is very helpful
for catching typos which happen quite frequently in practice, considering the
number of options that we have."

Reply:
I recommend to just pass --enable-option-checking (without arguments) to configure to fix the problem. The advantage is that configure succeeds, but you still get useful warnings: configure finishes with smth. like:

configure: WARNING: unrecognized options: --disable-static, --enable-doc, --enable-gtk-doc, --enable-gtk-doc-html, --enable-gtk-doc-pdf, --with-html-dir, --disable-doxygen